app/routes.py

```diff
@@ -1,2 +1,62 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
+
+
+class Planet():
+	def __init__(self, id, name, description, circumference, length_of_year):
+		self.id = id
+		self.name = name
+		self.description = description
+		self.circumference = circumference
+		self.length_of_year = length_of_year
+
+	def to_json(self): 
+		return {
+			"id" : self.id,
+			"name" : self.name,
+			"description" : self.description,
+			"circumference" : self.circumference,
+			"length_of_year" : self.length_of_year
+		}
+
+planets = [
+	Planet(1, "Mercury", "made mostly of rocks", 9522, 88), 
+	Planet(2, "Venus", "most like Earth", 23617, 225), 
+	Planet(3, "Earth", "you are here", 24889, 365),
+	Planet(4, "Mars", "the red planet", 13256, 687),
+	Planet(5, "Jupiter", "largest planet", 278985, 4320),
+	Planet(6, "Saturn", "sun's bae with all 7 rings", 235185, 10620),
+	Planet(7, "Uranus", "can only be seen with a telescope", 99739, 30240),
+	Planet(8, "Neptune", "it is an intense blue color", 96645, 59400),
+	Planet(9, "Pluto", "no dwarf in my book", 7144, 88920)
+]
+
+planets_bp = Blueprint("planets", __name__, url_prefix = "/planets")
+
+@planets_bp.route("", methods = ["GET"])
+def get_all_planets():
+	planets_response = []
+	for planet in planets:
+		planets_response.append({
+			"id": planet.id,
+			"name": planet.name,
+			"description": planet.description,
+			"circumference": planet.circumference,
+			"length of year": planet.length_of_year
+		})
+	return jsonify(planets_response)
+
+@planets_bp.route("/<id>", methods = ["GET"])
+def get_one_planet(id):
+	try:
+		id = int(id)
+	except:
+		return abort(make_response({"message":f"Planet {id} is invalid."}, 400))
+
+	for planet in planets:
+		if planet.id == id:
+			return jsonify(planet.to_json())
+	return abort(make_response({"message":f"Planet {id} not found."}, 404))
```

Comment:
Nice use of a try/except. The validation for bad id's and for id's with no record looks good. We can also consider moving the validation portion of this function into a helper method.